src/test/java/io/confluent/copycat/hdfs/HdfsSinkConnectorTestBase.java

```diff
@@ -99,15 +99,22 @@ protected GenericRecord createRecord() {
   @Before
   public void setUp() throws IOException {
     conf = new Configuration();
+    conf.setBoolean("dfs.client.block.write.replace-datanode-on-failure.enable", true);
```

Comment:
Are these important settings we'd need to advise use of or are they just for testing?